aip/general/0193.md

```diff
@@ -458,6 +340,85 @@ If the user does have proper permission, but the requested resource or
 parent does not exist, the service **must** error with `NOT_FOUND` (HTTP
 404).
 
+## HTTP/1.1+JSON representation
+
+When clients use HTTP/1.1 as per [AIP-127](./0127.md), the error information
+is returned in the body of the response, as a JSON object. For backward
+compatibility reasons, this does not map precisely to `google.rpc.Status`,
+but contains the same core information. The schema is defined in the following proto:
+
+```proto
+message Error {
+  message Status {
+    // The HTTP status code that corresponds to `google.rpc.Status.code`.
+    int32 code = 1;
+    // This corresponds to `google.rpc.Status.message`.
+    string message = 2;
+    // This is the enum version for `google.rpc.Status.code`.
+    google.rpc.Code status = 4;
+    // This corresponds to `google.rpc.Status.details`.
+    repeated google.protobuf.Any details = 5;
+  }
+
+  Status error = 1;
+}
+```
+
+The most important difference is that the `code` field in the JSON is an HTTP status code,
+*not* the direct value of `google.rpc.Status.code`. For example, a `google.rpc.Status`
+message with a `code` value of 5 would be mapped to an object including the following
+code-related fields (as well as the message, details etc):
+
+```json
+{
+  "error": {
+    "code": 404,          // The HTTP status code for "not found"
```

Comment:
How did you get coloring in here? Cool - not sure if it will render in AIP site generator

Reply:
TIL haha